execute/transport.go

```diff
@@ -360,3 +364,71 @@ func (m *finishMsg) Type() MessageType {
 func (m *finishMsg) Error() error {
 	return m.err
 }
+
+// consecutiveTransportTable is a flux.Table that is being processed
+// within a consecutiveTransport.
+type consecutiveTransportTable struct {
+	transport *consecutiveTransport
+	tbl       flux.Table
+}
+
+func newConsecutiveTransportTable(t *consecutiveTransport, tbl flux.Table) flux.Table {
+	return &consecutiveTransportTable{
+		transport: t,
+		tbl:       tbl,
+	}
+}
+
+func (t *consecutiveTransportTable) Key() flux.GroupKey {
+	return t.tbl.Key()
+}
+
+func (t *consecutiveTransportTable) Cols() []flux.ColMeta {
+	return t.tbl.Cols()
+}
+
+func (t *consecutiveTransportTable) Do(f func(flux.ColReader) error) error {
+	return t.tbl.Do(func(cr flux.ColReader) error {
+		if err := t.validate(cr); err != nil {
+			logger := t.transport.logger
+			logger.Info("Invalid column reader received from parent",
+				zap.String("source", t.transport.sourceInfo()),
```

Comment:
What exactly is included in the sourceInfo here?

Reply:
It's the line number information of the transformation that processed the bad table. So if you have `distinct() |> max()` and distinct produces a bad table, it will print `max` and the line number.

Reply:
And this will show up in the query log, so we'll have a way of identifying what query it came from, correct?

Reply:
It will but we'll have to search through where the query came from. I'm going to try out another change and push it up. Brett added the ability to access the jaeger trace id recently and I think I can probably use the same code path.

Reply:
I'm double checking with @brettbuddin but I think this will work. According to him in a DM, it should only work in the cloud product so it doesn't work if you try it in InfluxDB OSS.